CHANGELOG.md

```diff
@@ -40,6 +40,7 @@ Note that this project **does not** adhere to [Semantic Versioning](https://semv
 - We added automatic browser extension install on Windows for Chrome and Edge. [#6076](https://github.com/JabRef/jabref/issues/6076)
 - We added a search bar for filtering keyboard shortcuts. [#11686](https://github.com/JabRef/jabref/issues/11686)
 - By double clicking on a local citation in the Citation Relations Tab you can now jump the linked entry. [#11955](https://github.com/JabRef/jabref/pull/11955)
+- When dragging and dropping several PDF files into the Main Table, the BackgroundTask menu icon becomes a progress indicator visualising the import's progress. [#12035](https://github.com/JabRef/jabref/pull/12035)
```

Comment:
Changelog entries are for the end user. No technical terms (also no technical casing). OMG, while reviewing I found, that "Background Task" appears in the UI, too. Needs to be fixed (I filed an internal issue https://github.com/JabRef/jabref-issue-melting-pot/issues/644).

```suggestion
- When dragging and dropping several PDF files into the main table, the menu icon for background tasks becomes a progress indicator visualising the import's progress. [#12035](https://github.com/JabRef/jabref/pull/12035)
```